src/tasks/service.rs

```diff
@@ -509,17 +504,14 @@ impl ProxyService {
     ))]
     async fn handle_invoke_proxy_message_v1(
```

Comment:
I think you can even delete the entire function there, and when stuff fails to compile, just continue deleting everything that mentioned this code :)

Reply:
I decided to keep it, since the message "invoke proxy v1" message still exists.

Reply:
Actually it doesn't look like this is caused by a specific message. Will see how I can change it.

Reply:
Implemented in cae96d4da512658dfd05b27011f292f9d3838124